app/app.go

```
@@ -264,9 +326,23 @@ func (app *App) CallManager() call_manager.CallManager {
 	return app.callManager
 }
 
+// Ready reports whether this node can take traffic, per the health registry.
 func (app *App) Ready() (bool, model.AppError) {
-	//TODO
-	return true, nil
+	if app.health == nil {
+		return false, model.NewInternalError("app.ready.no_registry", "health registry is not initialised")
```

golangci-lint (2.12.2)

[error] `initialised` is a misspelling of `initialized`

(misspell)